src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/helpers/ReferralHelper.java

```diff
@@ -104,7 +109,18 @@ public static String populateHearingDetails(CaseData caseData) {
         if (CollectionUtils.isEmpty(caseData.getHearingCollection())) {
             return "";
         }
-        String trackType = caseData.getConciliationTrack();
+
+        String trackType = "";
+        if (caseData.getConciliationTrack() != null) {
+            if (caseData.getConciliationTrack().equals(NO_TRACK_OLD_NAME)) {
+                trackType = NO_TRACK_NAME;
+            } else if (caseData.getConciliationTrack().equals(SHORT_TRACK_OLD_NAME)){
+                trackType = SHORT_TRACK_NAME;
+            } else {
+                trackType = caseData.getConciliationTrack();
+            }
+        }

```

Comment:
Better to extract into it's own method.  This method is getting too big

Reply:
In fact, better to also switch this all around as you don't need an extra if here.  I.e. something like this:
`if(NO_TRACK_OLD_NAME.equals(caseData.getConciliationTrack()) {`